docs/manual/java/guide/production/AkkaDiscoveryIntegration.md

```diff
@@ -26,12 +26,35 @@ The Guice module [AkkaDiscoveryServiceLocatorModule](api/index.html?com/lightben
 
 In development, your Lagom application will keep using the Lagom's dev-mode [ServiceLocator](api/index.html?com/lightbend/lagom/javadsl/api/ServiceLocator.html).
 
-By default, Lagom uses [Aggregate multiple discovery methods](https://doc.akka.io/docs/akka/2.6/discovery/index.html#discovery-method-aggregate-multiple-discovery-methods). The first discovery method is set to Configuration and the second is set to DNS.
-So the static definition of service endpoins has a priority over DNS discovery.
+By default, Lagom uses [Aggregate multiple discovery methods](https://doc.akka.io/docs/akka/2.6/discovery/index.html#discovery-method-aggregate-multiple-discovery-methods). The first discovery method is set to Configuration and the second is set to DNS. So the static definition of service endpoints has a priority over DNS discovery.
 
 To statically configure service endpoints in your `application.conf` file consult the [Aggregate multiple discovery methods](https://doc.akka.io/docs/akka/2.6/discovery/index.html#discovery-method-aggregate-multiple-discovery-methods) documentation.
 
-To configure service discovery with DNS Lagom provides some configuration settings. The default settings are in
+## DNS SRV vs. DNS A/AAAA Lookups
+
+`AkkaDiscoveryServiceLocator` supports DNS SRV as well as DNS A lookups. It defaults to SRV lookups since it's the most common usage in environments like Kubernetes and DC/OS (Marathon).
```

Comment:
(nitpick)

```suggestion
`AkkaDiscoveryServiceLocator` supports DNS SRV as well as DNS A lookups. It defaults to SRV lookups favouring client-side balancing. DNS A lookups are useful when the remote service sits behind a load balancer and publishes a single location point (URL).
```

Reply:
I'm not so convinced about this change. I don't think we should justify why this is like that. We will be adding extra info to the docs. 

The reason we adopt this pattern is that it the most common. Why it became the most common doesn't fit in our docs.

Reply:
Seems like a good addition to me because: (a) I didn't know this, and I don't think it's because I'm so out of the loop, but because it's not well known, and (b) because it might provide useful for users who might be thinking in terms of load balancing only.

Reply:
Also, "The reason we adopt this pattern is that it the most common" sounds like appeal to tradition/common practice.